automatic/bolt/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,20 @@
+﻿$packageName = 'puppet-bolt'

```

Comment:
The package name here needs to match the directory name and nuspec filename, i.e. `bolt`